bin/email_notifications.js

```diff
@@ -31,7 +31,7 @@ var deliveryQueue = new SQSReceiver(config.emailNotifications.region, [
 ])
 
 DB.connect(config[config.db.backend])
-  .done(
+  .then(
```

Comment:
@philbooth What was the reason for this?, seems like http://bluebirdjs.com/docs/api/done.html is still supported